src/Runner.Worker/StepsRunner.cs

```diff
@@ -104,16 +104,7 @@ public async Task RunAsync(IExecutionContext jobContext)
                 if (step is IActionRunner actionStep)
                 {
                     // Set GITHUB_ACTION
-                    // Warning: Do not turn on FF DistributedTask.UseContextNameForGITHUBACTION until after M271-ish. After M271-ish
-                    // the server will never send an empty context name. Generated context names start with "__"
-                    if (step.ExecutionContext.Global.Variables.GetBoolean("DistributedTask.UseContextNameForGITHUBACTION") ?? false)
-                    {
-                        step.ExecutionContext.SetGitHubContext("action", actionStep.Action.Name);
-                    }
-                    else
-                    {
-                        step.ExecutionContext.SetGitHubContext("action", step.ExecutionContext.GetFullyQualifiedContextName());
-                    }
+                    step.ExecutionContext.SetGitHubContext("action", actionStep.Action.Name);
```

Comment:
reverting for now, will start over with a new FF name since the check was backwards